config/static/nodes/rpm-overrides/README.md

```diff
@@ -0,0 +1,56 @@
+# Setting the runc environment
+
+This configuration install into the pool indicated a custom runc version by
+installing a RPM package. By default it installs the package for "worker" and
+"master" nodes, but we can change that behavior.
+
+Basic usage:
+
+```shell
+export RPM_PACKAGES="https://..."
+make -C config/static/nodes/runc configure
+kustomize build config/static/nodes/runc | oc create -f -
+```
+
+For installing only in one specific pool:
+
+```shell
+export RPM_PACKAGES="https://..."
+export POOL="master"
+make -C config/static/nodes/runc configure
+kustomize build config/static/nodes/runc | oc create -f -
```

Comment:
Wrong reference into the paths, it should be: `config/static/nodes/rpm-overrides`.